public/app/features/explore/Explore.tsx

```diff
@@ -379,6 +391,18 @@ export class Explore extends React.PureComponent<ExploreProps, ExploreState> {
                               onStopScanning={this.onStopScanning}
                             />
                           )}
+                          {mode === ExploreMode.Tracing && (
+                            <div className={styles.fullHeight}>
+                              {queryResponse &&
+                                !!queryResponse.series.length &&
+                                queryResponse.series[0].fields[0].values.get(0) && (
```

Comment:
This looks suspicious. Why is this happening here? Seems like there is a very specific convention around the traces

Reply:
Well the datasource query should return dataframe so right now for traces it just puts all the trace data into first field first value. Don't have a good idea how to better map this to data frame tbh.